doc/examples/segmentation/plot_multiotsu.py

```diff
@@ -0,0 +1,79 @@
+"""
+=======================
+Multi-Otsu Thresholding
+=======================
+
+In this example we use multi-Otsu threshold to separate the pixels of an
+input image into three different classes, each one obtained according to
+the intensity of the gray levels within the image.
+
+Multi-Otsu calculates several thresholds, determined by the number of desired
+classes. The default number of classes is 3: for obtaining three classes, the
+algorithm returns two threshold values. They are represented by a red line in
+the histogram below.
+
+.. [1] Liao, P-S. and Chung, P-C., "A fast algorithm for multilevel
+thresholding", Journal of Information Science and Engineering 17 (5):
+713-727, 2001.
+"""
+
+from skimage.data import camera
+from skimage import img_as_float
+from skimage.filters import threshold_multiotsu
+
+import matplotlib
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+# Stablishing the font size for all plots.
+matplotlib.rcParams['font.size'] = 9
+
+# The input image.
+image = camera()
+
+# Applying multi-Otsu threshold for the default value, generating
+# three classes.
+thresh = threshold_multiotsu(image)
+print('Threshold values: ', thresh)
```

Comment:
I don't think we want a print here. However, you can display that on a histogram. You can use

```
ax.axvline(thresh[0], color='r')
```
for example